packages/core/contracts/common/implementation/DSProxyFactory.sol

```diff
@@ -0,0 +1,203 @@
+// This file was originally take from from Maker: DS Proxy Factory and modified. The original source code can be found
+// here: https://etherscan.io/address/0xa26e15c895efc0616177b7c1e7270a4c7d51c997#code
+// Changes are limited to updating the Solidity version and some stylistic modifications.
```

Comment:
To make reviewing easier can you comment where you made changes from the copied and battle-tested code?

Reply:
+1

Reply:
the only changes were updating the solidity version & the associated syntactical changes that changed between versions (emit on events, `memory` on bytes params) and linting the file to fit to our style guide.